.env.example

```
@@ -3,13 +3,16 @@ API_URL=
 GLIFIC_DOC_UUID=
 BOT_TOKEN=
 CHANNEL_ID=
+GUILD_ID=
 GCP_CLIENT_EMAIL=
 GCP_PRIVATE_KEY=
 SPREADSHEET_ID=
+OPENAI_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The OPENAI_API_KEY key is duplicated

(DuplicatedKey)